src/components/notifications/CallMeBot.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+    <div class="mb-3">
+        <label for="callmebot-endpoint" class="form-label">{{ $t("Endpoint") }}</label>
+        <div class="input-group mb-3">
+            <input id="callmebot-endpoint" v-model="$parent.notification.callMeBotEndpoint" type="text" class="form-control" required>
+            <button v-if="$parent.notification.callMeBotEndpoint" class="btn btn-outline-secondary" type="button" @click="cleanupEndpoint">
+                {{ $t("Cleanup") }}
+            </button>
+        </div>
+        <i18n-t tag="div" keypath="callMeBotGet" class="form-text">
+            <a href="https://www.callmebot.com/blog/free-api-facebook-messenger/" target="_blank">Facebook Messenger</a>
+        </i18n-t>
+        <i18n-t tag="div" keypath="callMeBotGet" class="form-text">
+            <a href="https://www.callmebot.com/blog/test-whatsapp-api/" target="_blank">WhatsApp</a>
+        </i18n-t>
+        <i18n-t tag="div" keypath="callMeBotGet" class="form-text">
+            <a href="https://www.callmebot.com/blog/telegram-phone-call-using-your-browser/" target="_blank">Telegram Call</a>
+        </i18n-t>
+        <i18n-t tag="div" keypath="callMeBotInfo" class="form-text" />
```

Comment:
This helptext is quite weird.
Think about a user coming across this: would they be able to figure out what to do in this case?

Please:
- refactor this into a single helptext which does not contain the same text 3x
- consider linking to the ratelimits instead of telling people what one of them has currently (this may change)

Reply:
Applied the suggestions in cb15ea0.
The information about the ratelimits is still missing, as this is not publicly documented.
I'm awaiting the response of the HelpMeBot support in order do give some more details about rate limits